pyproject.toml

```diff
@@ -80,7 +80,7 @@ pre-commit = "^2.14.0"
 autoflake = "^1.4"
 pyment = "^0.3.3"
 tox = "^3.24.3"
-hypothesis = "^6.23.0"
+hypothesis = "^6.43.1"
```

Comment:
Is the `HealthCheck` part of a later version?

Reply:
the healthcheck had also existed in the previous problem, but it was supressed by the pytest.fixture decorator. the newer version exposed the healthcheck and thus forced me to supress it in the settings decorator instead.

Reply:
Makes sense.